src/Jellyfin.XmlTv/XmlTvReader.cs

```diff
@@ -14,7 +14,7 @@ namespace Jellyfin.XmlTv
     // Reads an XmlTv file
     public class XmlTvReader
     {
-        private const string DateWithOffsetRegex = @"^(?<dateDigits>[0-9]{4,14})(\s(?<dateOffset>[+-]*[0-9]{1,4}))?$";
+        private const string DateWithOffsetRegex = @"^(?<dateDigits>[0-9]{4,14})(\s(?<dateOffset>[+-]*[0-9]{1,2}:{0,1}[0-9]{0,2}))?$";
```

Comment:
With this new regex, `+9:` is valid. I don't know if we really need to be super restrictive, but this is very loose 😄